plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -207,11 +226,30 @@ app_get_flatpak_ref_name (GsApp *app)
 	return gs_app_get_metadata_item (app, "flatpak::RefName");
 }
 
+static gchar **
+gs_plugin_get_default_locales (GsPlugin *plugin)
+{
+	g_auto(GStrv) flatpak_default_locales = NULL;
+	g_autoptr(GError) local_error = NULL;
+	g_autoptr(FlatpakInstallation) installations = NULL;
+
+	GsPluginData *priv = gs_plugin_get_data (plugin);
+
+	flatpak_default_locales = flatpak_installation_get_default_languages (priv->inst, &local_error);
```

Comment:
`priv->inst` could be `NULL` (if setup failed), which would result in a critical warning here.